<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>project_properties</class>
 <widget class="QDialog" name="project_properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>487</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>487</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project properties</string>
  </property>
  <layout class="QVBoxLayout" name="project_properties_layout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="general_layout">
       <item>
        <widget class="QGroupBox" name="general_info">
         <property name="title">
          <string>Project information</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="general_info_layout">
          <item row="0" column="0">
           <widget class="QLabel" name="project_name_label">
            <property name="text">
             <string>Project name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="project_name_edit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="author_label">
            <property name="text">
             <string>Author</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="version_label">
            <property name="text">
             <string>Version</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="author_edit"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="version_edit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="startup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Startup</string>
      </attribute>
      <layout class="QVBoxLayout" name="startup_layout">
       <item>
        <widget class="QGroupBox" name="startup_info">
         <property name="title">
          <string>Application startup</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="startup_info_layout">
          <item row="0" column="0">
           <widget class="QLabel" name="startup_scene_label">
            <property name="text">
             <string>Startup scene</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="startup_scene_container" native="true">
            <layout class="QHBoxLayout" name="startup_scene_container_layout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="plugins">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QVBoxLayout" name="plugins_layout">
       <item>
        <widget class="QGroupBox" name="plugin_loading">
         <property name="title">
          <string>Plugin loading</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="plugin_loading_layout">
          <item>
           <widget class="QWidget" name="plugins_platform_form" native="true">
            <layout class="QFormLayout" name="plugins_platform_form_layout">
             <item row="0" column="0">
              <widget class="QLabel" name="plugins_platform_label">
               <property name="text">
                <string>Platform</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="plugins_platforms">
               <item>
                <property name="text">
                 <string>Windows</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Linux</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="plugins_tip">
            <property name="text">
             <string>The following libraries will be loaded for the currently selected platform. The modules will be loaded in order.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="plugins_form" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="plugins_form_layout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListView" name="plugins_list"/>
             </item>
             <item>
              <widget class="QWidget" name="plugins_list_buttons" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="plugins_list_buttons_layout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="plugins_list_add">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="plugins_list_delete">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="plugins_list_up">
                  <property name="text">
                   <string>Up</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="plugins_list_down">
                  <property name="text">
                   <string>Down</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="plugins_list_spacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="button_container" native="true">
     <layout class="QHBoxLayout" name="button_container_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QDialogButtonBox" name="buttons">
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
